app/src/main/java/org/oppia/android/app/drawer/NavigationDrawerFragmentPresenter.kt

```diff
@@ -240,7 +239,12 @@ class NavigationDrawerFragmentPresenter @Inject constructor(
             fragment.childFragmentManager.beginTransaction().remove(previousFragment).commitNow()
           }
           val dialogFragment = ExitProfileDialogFragment
-            .newInstance(isFromNavigationDrawer = true)
+            .newInstance(
+              isFromNavigationDrawer = true,
+              isAdministratorControlsSelected =
+                getFooterViewModel().isAdministratorControlsSelected.get() ?: false,
```

Comment:
reformat code

Reply:
Done

Reply:
Actually, ktlint is causing an error here, so I think we shouldn't reformat here.